addons/controllers/errorfile

```diff
@@ -0,0 +1,173 @@
+Running Suite: Addon Controller Suite
```

Comment:
Delete file?

Reply:
Omg, how this one has been committed! Definitely, thanks for actually noticing it